cmd/pbapi/main.go

```diff
@@ -181,7 +181,7 @@ func main() {
 
 	if err := apiServer.ListenAndServe(); err != nil {
 		if !errors.Is(err, http.ErrServerClosed) {
-			log.Fatal().Err(err).Msg("Main service listen error")
+			log.Warn().Err(err).Msg("Main service listen error")
```

Comment:
I noticed that we call Fatal when there is a shutdown error. Fatal immediately exists the application will kill other goroutines. We should never use Fatal unless we want to exit immediately (e.g. in migrations or during startup).